src/main/resources/META-INF/rewrite/java-version-20.yml

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2022 the original author or authors.
```

Comment:
Nitpick: You seem to have an outdated copyright setting in IDEA (or copied it from somewhere).